main-net/terraform.tfvars

```diff
@@ -0,0 +1,3 @@
+cluster_info_url = "http://cl-lb-alb-testnet-118182741.us-west-1.elb.amazonaws.com:9000/cluster/info"
```

Comment:
can we take tfvars out of the repository? preferable keybase as we did for network/infrastructure